Package.swift

```diff
@@ -139,4 +139,8 @@ testUtilityTarget.swiftSettings.append(contentsOf: combineImp.swiftSettings)
 
 if combineImp == .combine && isCI {
     package.platforms = [.macOS("10.15"), .iOS("13.0"), .tvOS("13.0"), .watchOS("6.0")]
+} else {
+    #if swift(>=5.3)
```

Comment:
Probably `compiler(>=5.3)`? I know it doesn't actually make a difference (CombineX doesn't support swift 4.2 mode), but logically it's not really about swift itself.

Reply:
That would be a lot better 👍